Makefile

```diff
@@ -22,7 +22,7 @@ BUILD_DIR := _output
 # VERSION is derived from the current tag for HEAD plus amends. Version is used
 # to set/overide the CRIContainerdVersion variable in the verison package for
 # cri-containerd.
-VERSION := $(shell git describe --tags --dirty)
+VERSION := $(shell git describe --tags --dirty --always)
```

Comment:
I'm fine with keeping `always` here. But we should keep the unit test unchanged, so that we can catch this issue again in the futrue

Reply:
Not really an issue more a feature.. shouldn't break a pr or private build due missing tag...

Just have to remember to push a tag when you are going to produce a new release. And if you want to see the new release tag on your clone you have to fetch --tags.

Reply:
The unit test is just unit testing whether we can semver the version tag it's not unit testing whether the tag is correct or ensuring the version is only a tag.  I'm thinking this is useful for local developers.

Reply:
At one point unknown was the default.. dunno. I like it this way :-)

Reply:
@mikebrow The version will be used by our CI verison of containerd as well. And kubernetes does require the version to be semver.

Reply:
done.